src/lib/ngsi/ContextAttribute.cpp

```diff
@@ -596,7 +596,7 @@ std::string ContextAttribute::toJson(bool isLastElement, bool types, const std::
     {
       if (type == DATE_TYPE)
       {
-        out += isodate2str(numberValue);
+        out += JSON_STR(isodate2str(numberValue));
```

Comment:
It is a bug fix, but intra-release. I mean, it is covered by the current CNR, in particular in this entry:

"Add: dates support in attribute values (Issue #1038)"